geocruncher/main.py

```diff
@@ -64,6 +64,19 @@ def run_geocruncher(args):
         # TODO do something useful with output files
         print(generated_mesh_paths)
 
+    if args[1] == 'slice':
+        nPoints=500
+        with open(args[2]) as f:
+            data = json.load(f)
+        for rect in data:
+            xCoord=[rect["lowerLeft"]["x"], rect["upperRight"]["x"]]
+            yCoord=[rect["lowerLeft"]["y"], rect["upperRight"]["y"]]
+            zCoord=[rect["lowerLeft"]["z"], rect["upperRight"]["z"]]
+            outputs = {'forSurface': CrossSectionSlice.output(xCoord,yCoord,zCoord,nPoints,model, [1, 1])} 

```

Comment:
I have renamed the class to ``Slice``, so this needs to be fixed. I think this "merge" here https://github.com/ISSKA/geocruncher/pull/13/commits/e17bbc0be96ca1cf0ac91b8dacaa8ba0128efc10 went wrong.

Reply:
yes you are right, currently doing the test case BTW